src/modules/events/events.js

```diff
@@ -29,3 +30,11 @@ bus.on('change', (change) => {
 		else winston.info('Events registered');
 	});
 });
+
+bus.on('http/init', app => {
+	app.use(route.post('/x/change', function *() {
```

Comment:
@harishkumarv nobody ensures that this change event has a session id, and it's not any different from any other change events emitted in server, which don't need a session id.